compiler/src/iree/compiler/Codegen/Sandbox/Passes.h

```diff
@@ -38,6 +38,10 @@ std::unique_ptr<OperationPass<func::FuncOp>> createLinalgFusePass();
 std::unique_ptr<OperationPass<func::FuncOp>> createLinalgFusePass(
     const LinalgFusePassOptions &options);
 
+/// Creates a pass to perform SplitReduction transformations of `LinalgOp`s.
+std::unique_ptr<OperationPass<func::FuncOp>> createLinalgSplitReductionPass(
+    const bool enableReassociateFpReductions = false, const int64_t size = 0);
```

Comment:
please drop meaningless `const`. (a) it's not consistent with others. (b) They are meaningless in function declaration. See https://abseil.io/tips/109 for more details.

```suggestion
   bool enableReassociateFpReductions = false, int64_t size = 0);
```

Reply:
Done